### PR TITLE
Add Basic Page Model And Page Not Found

### PR DESCRIPTION
**Pull request by Lordapa**

### CODE REVIEW

File: src/DonateTo.WebApplication/ClientApp/i18n/en.json [hunk 1]
```diff
@@ -75,8 +75,8 @@
   },
   "PageNotFound": {
     "Title": "Ooops, Page Not Found",
-    "Subtitle": "Please, return to the previous page",
-    "Button": "Go back"
+    "Subtitle": "La página que busca no existe o se produjo otro error.",
```
Comment (nicolasgranata) [inline src/DonateTo.WebApplication/ClientApp/i18n/en.json:78]: Should be in english

File: src/DonateTo.WebApplication/ClientApp/src/app/basic-pages/basic-pages.module.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { NgModule } from '@angular/core';
+import { IconDefinition } from '@ant-design/icons-angular';
+
+import { FrownOutline } from '@ant-design/icons-angular/icons';
+
+import { NzButtonModule, NzIconModule, NzLayoutModule } from 'ng-zorro-antd';
+
+import { TranslateLoader, TranslateModule } from '@ngx-translate/core';
+import { HttpLoaderFactory } from 'src/app/app.module';
```
Comment (facundo-lewis) [inline src/DonateTo.WebApplication/ClientApp/src/app/basic-pages/basic-pages.module.ts:9]: I do not like importing anything from app.module. For this instance, copy that method. 
We will add this function as a utility

File: src/DonateTo.WebApplication/ClientApp/i18n/en.json [hunk 3]
```diff
@@ -75,8 +75,8 @@
   },
   "PageNotFound": {
     "Title": "Ooops, Page Not Found",
-    "Subtitle": "Please, return to the previous page",
-    "Button": "Go back"
+    "Subtitle": "The page you are looking for does not exist or another error occurred.",
```
Comment (Faculewis) [inline src/DonateTo.WebApplication/ClientApp/i18n/en.json:78]: Remove "or other error occurred", that message confuses the user. This error is to indicate to the users that the page they requested was not found, if there is another error we must be specific with what happened.

File: src/DonateTo.WebApplication/ClientApp/src/app/basic-pages/not-found/not-found.component.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<nz-layout>
+  <nz-content>
+    <i nz-icon [nzTheme]="'outline'" nzType="frown"></i>
+    <h1>404</h1>
```
Comment (Faculewis) [inline src/DonateTo.WebApplication/ClientApp/src/app/basic-pages/not-found/not-found.component.html:4]: Change the h1 to another smaller font size. Users don't know what a 404 means and it's irrelevant to them.
The message we show must be accurate and friendly. Take a look to other services how do they show that error, for example:
- https://www.google.com/test
- https://www.microsoft.com/test

**Pull request merged**